digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/LaneUtils.scala

```diff
@@ -72,6 +73,27 @@ object LaneUtils {
 
   }
 
+
+  def fixSideCode( roadAddress: RoadAddressesAux , laneCode: String ): SideCode = {
+    roadAddress.track.value match {
+      case 1 | 2 => SideCode.BothDirections // This means the road may have both ways with something between them (like highways or similar)
+                                            // The representation of the lane will be in the middle 'of the road'
+
+      case _ => roadAddress.sideCode match { // In this case the road have both ways 'connected' so we need to take attention of the SideCode and laneCode
+                                             // This will have influence in representation of the lane
+                    case SideCode.AgainstDigitizing => if (laneCode.startsWith("1")) SideCode.AgainstDigitizing
+                                                        else SideCode.TowardsDigitizing
+
+                    case SideCode.TowardsDigitizing => if (laneCode.startsWith("1")) SideCode.TowardsDigitizing
+                                                        else SideCode.AgainstDigitizing
+
+                    case _ => SideCode.BothDirections
+                    }
+    }
+
+  }
```

Comment:
Move this to LaneService please.

Reply:
LaneFiller can't reach LaneUtils neither LaneService

Reply:
Moved as requested